tutorials/Application.getWindow.md

```diff
@@ -0,0 +1,20 @@
+Returns an instance of the main Window of the application
```

Comment:
Keep in mind that there will be quite a few of these (one per API call) so I would like the example to exclude things like the `require` statement and the call to `connect`. Lets assume we are in an environment with both and only focus on the example: 

```
async function getAppWindowExample() {
 		const app = await fin.Application.create({ uuid, name: "myApp" });
 		await app.run();
 		return await app.getWindow();
 }

getAppWindowExample().them(win => console.log(win));
```
Also Lets get some semicolons in here `;`.